controllers/example.js

```diff
@@ -21,19 +23,13 @@ exports.postExample = async (req, res) => {
 
 exports.deleteExample = async (req, res) => {
   const exampleId = req.params.exampleId
-  try {
-    const example = await Example.findById(exampleId)
+  const example = await Example.findById(exampleId)
 
-    if (!example) {
-      const error = new Error('Could not find example.')
-      error.statusCode = 404
-      throw error
-    }
+  if (!example) {
+    throw createError(404, EXAMPLE_NOT_FOUND)
+  }
 
-    await Example.findByIdAndRemove(exampleId)
+  await Example.findByIdAndRemove(exampleId)
 
-    res.status(200).json({ message: 'Example deleted.' })
-  } catch (e) {
-    res.status(e.statusCode).json(e.message)
-  }
+  res.status(200).json({ message: 'Example deleted.' })
```

Comment:
didn't we agree to send `res.status(204).end()` , did we?

Reply:
fixed